### ISSUE TITLE
 Error in load_data_detection() - OpenCV 

### ISSUE THREAD

**Issue by hichamhendy** [AlexeyAB/darknet#8664]
I am getting the error ` Error in load_data_detection() - OpenCV ` on Linux rnd-Intel-X299-Overclocked 5.15.0-46-generic #49~20.04.1-Ubuntu SMP Thu Aug 4 19:15:44 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux,

I think the error comes especially if train with multiple GPUs, using the latest commit
The Error
` Error in load_data_detection() - OpenCV 
v3 (iou loss, Normalizer: (iou: 0.05, obj: 0.40, cls: 0.50) Region 139 Avg (IOU: 0.902482), count: 13, class_loss = 0.242014, iou_loss = 38.210827, total_loss = 38.452843 
v3 (iou loss, Normalizer: (iou: 0.05, obj: 0.40, cls: 0.50) Region 150 Avg (IOU: 0.000000), count: 1, class_loss = 0.002116, iou_loss = 0.003173, total_loss = 0.005289 
v3 (iou loss, Normalizer: (iou: 0.05, obj: 0.40, cls: 0.50) Region 161 Avg (IOU: 0.000000), count: 1, class_loss = 0.000101, iou_loss = 0.000152, total_loss = 0.000253 
 total_bbox = 740093, rewritten_bbox = 0.463320 % 
v3 (iou loss, Normalizer: (iou: 0.05, obj: 0.40, cls: 0.50) Region 139 Avg (IOU: 0.000000), count: 1, class_loss = 0.029174, iou_loss = 0.043762, total_loss = 0.072936 
v3 (iou loss, Normalizer: (iou: 0.05, obj: 0.40, cls: 0.50) Region 150 Avg (IOU: 0.824206), count: 1, class_loss = 0.112110, iou_loss = 0.305635, total_loss = 0.417746 
v3 (iou loss, Normalizer: (iou: 0.05, obj: 0.40, cls: 0.50) Region 161 Avg (IOU: 0.400721), count: 1, class_loss = 0.016904, iou_loss = 0.034813, total_loss = 0.051717 
 total_bbox = 740095, rewritten_bbox = 0.463319 % 
Segmentation fault (core dumped)`

```
CUDA-version: 11070 (11070), cuDNN: 8.5.0, GPU count: 2  
 OpenCV version: 4.2.0
 0 : compute_capability = 750, cudnn_half = 0, GPU: NVIDIA GeForce RTX 2080 Ti 
net.optimized_memory = 0 
mini_batch = 1, batch = 32, time_steps = 1, train = 0 

```

![image](https://user-images.githubusercontent.com/45830803/188840547-31136336-3c10-48c1-b8b2-f697e829f14f.png)

![image](https://user-images.githubusercontent.com/45830803/188801223-ace03f0a-a0a9-4b6a-9b59-2e7567020cb0.png)

**Comment by dafanqie-an** [comment]
Have you solved it?Ihave the same question
